bundles/org.openhab.binding.mybmw/src/main/java/org/openhab/binding/mybmw/internal/console/MyBMWCommandExtension.java

```diff
@@ -314,7 +314,8 @@ public boolean complete(String[] args, int cursorArgumentIndex, int cursorPositi
                         .filter(t -> THING_TYPE_CONNECTED_DRIVE_ACCOUNT.equals(t.getThingTypeUID())
                                 && args[1].equals(t.getConfiguration().get("userName")))
                         .map(t -> t.getHandler()).findAny().get();
-                List<VehicleBase> vehicles = handler.getMyBmwProxy().get().requestVehiclesBase();
+                List<VehicleBase> vehicles = handler != null ? handler.getMyBmwProxy().get().requestVehiclesBase()
+                        : new ArrayList<>();
```

Comment:
```suggestion
                        : List.of();
```

Reply:
@mherwege looks like now the build is broken 😀 will fix it...